src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue/Services/Extensions/CommitmentExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.ProviderPayments.Calc.PaymentsDue.Domain;
+
+namespace SFA.DAS.ProviderPayments.Calc.PaymentsDue.Services.Extensions
+{
+    public static class CommitmentExtensions
```

Comment:
can be put onto the type and then tested.

Reply:
It is tested. What else are you after?